cm-core/src/main/java/ch/racic/testing/cm/ConfigProvider.java

```diff
@@ -238,6 +238,9 @@ public String get(String key, String defaultValue) {
         } else if (propsGlobal != null && propsGlobal.containsKey(key)) {
             log.debug("Retrieved property [" + key + "] from Global properties");
             return propsGlobal.getString(key);
+        } else if (systemProperties != null && systemProperties.containsKey(key)) {
```

Comment:
Order from the logic should be that system properties override everything in the inside of the props chain